refactor/results/app/core/config.py

```diff
@@ -30,6 +30,7 @@
 class Settings(BaseSettings):
     API_V0_STR: str = "/api/v0"
     BACKEND_CORS_ORIGINS: List[AnyHttpUrl] = []
+    RESULTS_SVC_PORT: int = 8006
```

Comment:
Note: The `BaseSettings` class from pydantic (https://pydantic-docs.helpmanual.io/usage/settings/) loads attributes within Settings class as environment variables under the hood